src/tool_getparam.c

```diff
@@ -1387,10 +1417,18 @@ ParameterError getparameter(const char *flag, /* f or -long-flag */
           char *enc = curl_easy_escape(NULL, postdata, (int)size);
           Curl_safefree(postdata); /* no matter if it worked or not */
           if(enc) {
+            size_t outlen;
+            char *n;
+            char *reenc = replace_url_encoded_space_with_plus(enc);
+            curl_free(enc);
+            if(!reenc)
+              return PARAM_NO_MEM;
+            enc = reenc;
```

Comment:
Note in this case you must now free instead of curl_free at 1444. (I can't put the comment there because it's not part of the changes.)

Reply:
oops